jetstream/fakespot-in-suggest.toml

```diff
@@ -199,12 +229,55 @@ select_expression = "COUNTIF(event_action = 'annoyance' and engaged_result_type
 data_source = "urlbar_events"
 [metrics.fakespot_not_relevant_dismissals.statistics.bootstrap_mean]
 
-## Review Checker
-[metrics.review_checker_displayed_count]
-select_expression = "COUNTIF(event_category = 'shopping' and event.name = 'surface_displayed')"
-data_source = "events"
-[metrics.review_checker_displayed_count.statistics.bootstrap_mean]
+
+[metrics.fakespot_4_star_A_grade]
+select_expression = "COUNTIF(mozfun.map.get_key(event_extra, 'grade') = 'A' and (CAST(mozfun.map.get_key(event_extra, 'rating') AS FLOAT64) >= 4 and CAST(mozfun.map.get_key(event_extra, 'rating') AS FLOAT64) < 5))"
```

Comment:
I recommend moving the logic for `grade` and `rating` (pulling from the map, casting) to the dataset query so that it doesn't need to be repeated each time.